.github/workflows/tests.yml

```diff
@@ -103,6 +103,20 @@ jobs:
           st_download_data prelude
           sudo install prelude/prelude /usr/local/bin
 
+      - name: FSL macOS
+        if: contains(matrix.os, 'macos')
+        run: |
+          echo "Download FSL for macOS"
+          source ~/shimming-toolbox/python/bin/activate
+          conda install -c https://fsl.fmrib.ox.ac.uk/fsldownloads/fslconda/public/ -c conda-forge fsl-bet2
+      
+      - name: FSL Ubuntu
+        if: contains(matrix.os, 'ubuntu')
+        run: |
+          echo "Download FSL for macOS"
+          source ~/shimming-toolbox/python/bin/activate
+          conda install -c https://fsl.fmrib.ox.ac.uk/fsldownloads/fslconda/public/ -c conda-forge fsl-bet2
```

Comment:
If this is the same for both macOS and Ubuntu, merge as one step and remove the "if"